### PR TITLE
Add lawn mower entity

### PR DESCRIPTION
**Pull request by mikey0000**
## Proposed change

Add documentation for the lawn mower entity

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

Link to core integration pull request: https://github.com/home-assistant/core/pull/93623
Link to brands pull request: https://github.com/home-assistant/brands/pull/4561
Link to frontend pull request: https://github.com/home-assistant/frontend/pull/17558
Link to developer documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1875

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lawn_mower.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Lawn Mower
+description: Instructions on how to setup and use lawn mowers in Home Assistant.
+ha_release: 2023.10
```
Comment (frenck) [inline source/_integrations/lawn_mower.markdown]:
```suggestion
ha_release: 2023.9
```

File: source/_integrations/lawn_mower.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ha_domain: lawn_mower
+ha_quality_scale: internal
+ha_category:
+  - Lawn Mower
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+The `lawn_mower` integration enables the ability to control home lawn mowers within Home Assistant.
```
Comment (frenck) [inline source/_integrations/lawn_mower.markdown]: I suggest taking a quick look at the [event documentation](https://www.home-assistant.io/integrations/event), as that provides the most recent way to approach this.

We need to make it really clear this is not something you can create yourself, but is an entity that is provided by other integrations.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/lawn_mower.markdown (hunk 2 above)
Comment (mikey0000) [reply source/_integrations/lawn_mower.markdown]: Ah yes, I'll take a look asap in the morning.

Comment (mikey0000) [reply source/_integrations/lawn_mower.markdown]: Can you please review the top sentence as well, I think it is an overall better sentence but open to changing it back.

File: source/_integrations/lawn_mower.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ha_domain: lawn_mower
+ha_quality_scale: internal
+ha_category:
+  - Lawn Mower
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+The `lawn_mower` integration allows the control of robotic lawn mowers to be reflected within Home Assistant.
```
Comment (frenck) [inline source/_integrations/lawn_mower.markdown]:
```suggestion
The lawn mower integration allows the control of robotic lawn mowers to be reflected within Home Assistant.
```

**Review by frenck — APPROVED** [review]
Thanks, @mikey0000 👍 

../Frenck

**Pull request merged**